frontend/components/badge-examples.js

```diff
@@ -30,47 +30,30 @@ export default class BadgeExamples extends React.Component {
   static propTypes = {
     definitions: PropTypes.array.isRequired,
     baseUrl: PropTypes.string,
-    longCache: PropTypes.bool.isRequired,
     onClick: PropTypes.func.isRequired,
   }
 
   renderExample(exampleData) {
-    const { baseUrl, longCache, onClick } = this.props
+    const { baseUrl, onClick } = this.props
     const { title, example, preview } = exampleData
 
-    let previewUrl
-    // There are two alternatives for `preview`. Refer to the schema in
-    // `services/service-definitions.js`.
-    if (preview.label !== undefined) {
-      const { label, message, color, style, namedLogo } = preview
-      previewUrl = staticBadgeUrl({
-        baseUrl,
-        label,
-        message,
-        color,
-        style,
-        namedLogo,
-      })
-    } else {
-      const { path, queryParams } = preview
-      previewUrl = badgeUrlFromPath({ baseUrl, path, queryParams, longCache })
-    }
+    const { label, message, color, style, namedLogo } = preview
+    const previewUrl = staticBadgeUrl({
```

Comment:
QQ for myself, where/how is this used with static examples and the modal window? (or perhaps I should ask the question on the `exampleUrl` below as I'm not sure what the difference is anymore

Reply:
This function renders a row in the badge listings. The `handleClick` function defined in here opens the modal. The row contains three columns.

The title:

<img width="86" alt="screen shot 2019-01-30 at 5 59 50 pm" src="https://user-images.githubusercontent.com/1487036/52020983-e0772d80-24b8-11e9-89ce-de4b4aaa3cf1.png">

`<ClickableTh onClick={handleClick}>{title}:</ClickableTh>`

The preview badge:

<img width="107" alt="screen shot 2019-01-30 at 5 59 52 pm" src="https://user-images.githubusercontent.com/1487036/52020997-f553c100-24b8-11e9-8dcf-d3f6c46b713b.png">

```js
<td>
  <Badge clickable onClick={handleClick} src={previewUrl} />
</td>
```

The example pattern:

<img width="425" alt="screen shot 2019-01-30 at 5 59 54 pm" src="https://user-images.githubusercontent.com/1487036/52021000-f97fde80-24b8-11e9-8e19-79ea39975dc0.png">

```js
<td>
  <ClickableCode onClick={handleClick}>{exampleUrl}</ClickableCode>
</td>
```

Does that help?

Reply:
Yeah, a lot actually thanks! I swear one of these days I'm going to find and read a React for dummies 😄 That `ClickableCode` element sounds pretty sweet